ci/deps/azure-36-minimum_versions.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
 
   # tools
   - cython=0.29.16
-  - pytest=5.0.1
+  - pytest>=5.0.1, <6.0.0rc0
```

Comment:
@TomAugspurger this is minimum versions. should the pin be kept here

Reply:
Ah, yeah it should be pinned exactly.

Reply:
submitted PR #35274